relay_test.go

```diff
@@ -384,3 +384,49 @@ func TestRelayOutgoingConnectionsEphemeral(t *testing.T) {
 		require.NoError(t, testutils.CallEcho(ts.Server(), ts.HostPort(), "s2", nil), "CallEcho failed")
 	})
 }
+
+func TestRelayHandleLocalCall(t *testing.T) {
+	opts := testutils.NewOpts().SetRelayOnly().
+		SetRelayLocal([]string{"relay", "tchannel", "test"}).
+		// We make a call to "test" for an unknown method.
+		AddLogFilter("Couldn't find handler.", 1)
+	testutils.WithTestServer(t, opts, func(ts *testutils.TestServer) {
+		s2 := ts.NewServer(serviceNameOpts("s2"))
+		testutils.RegisterEcho(s2, nil)
+
+		client := ts.NewClient(nil)
+		require.NoError(t, testutils.CallEcho(client, ts.HostPort(), "s2", nil), "CallEcho to relayed service failed")
```

Comment:
Any reason not to use `AssertEcho` here too? I know it does a bit more than `CallEcho`, but none of the extra checks seem inappropriate here.